### ISSUE TITLE
what's the dynamic contenet? 

### ISSUE THREAD

**Issue by Maxhodges** [ishan-me/JAMStack_with_Gridsome#1]
Hi I read your medium post about this, but is there really "dynamic" content? or it's all just static (pre-rendered on build)?

I'm looking for a Gridsome JAMstack examples that actually had dynamic content (not just pre-build, static site)..

Cheers,

**Comment by isNan909** [comment]
Yes we do have dynamic content for our application as well. But in the article I have used markdown which serves content via GraphQL layer. 

We are using a Static Site generator but also can be taken as making a JAMStack application with third party API for our application as well.
